archive/use-cases.html

```diff
@@ -139,6 +144,28 @@
      Other use cases may include bulk transfer of contacts cards, photos,
      and other data.
     </p>
+    <p class="note">
+      This use case is not supported in this version of the specification.
+    </p>
+  </section>
+  <section><h3>Payment scenarios</h3>
+    <p>
+      Payment scenarios with Web NFC generally do not refer to supporting
```

Comment:
Payment using NFC should be handled at a platform level due to security considerations and is thus not within the scope of Web NFC. On the other hand it is possible to associate...